pyproject.toml

```diff
@@ -56,6 +54,18 @@ mkdocs-gen-files = "^0.4.0"
 mkdocs-literate-nav = "^0.5.0"
 mkdocs-section-index = "^0.3.4"
 
+[tool.ruff]
+target-version = "py312"
```

Comment:
We are meant to support from python 3.9, so target version should be that one. Otherwise, there will be functionality that is not backwards compatible.

Reply:
Yep, sorry, overlooked that one